Gemfile.lock

```diff
@@ -55,7 +55,7 @@ GEM
       procto (~> 0.0.2)
       unparser (~> 0.5.3)
       variable (~> 0.0.1)
-    mutant-license (0.1.0)
+    mutant-license (0.1.1.2.2355046999240944981729280251890364410689.0)
```

Comment:
For anyone interested:

`0.1.1` is the "real" version.
`.2` is the encoding of oss license.
`.2355046999240944981729280251890364410689` is the decimal represenation of the account UUID (4x word32).
`.0` is the revision (no revision since I made the mechanism, will increase which each repository/author added/removed from the license).